examples/00-mapdl-examples/composite_dcb.py

```diff
@@ -452,6 +452,6 @@
 # Exit MAPDL
 try:
     os.remove(rst_path)
-except:
+except FileNotFoundError or PermissionError:
```

Comment:
This is the way:

```suggestion
except (FileNotFoundError, PermissionError) as e:
```

https://stackoverflow.com/a/6470452/6650211